v0/destinations/marketo/transform.js

```diff
@@ -93,24 +104,27 @@ const createOrUpdateLead = async (
       action: "create"
     });
     const { accountId } = formattedDestination;
-    const resp = await sendPostRequest(
-      `https://${accountId}.mktorest.com/rest/v1/leads.json`,
-      // `https://httpstat.us/200`,
-      {
-        action: "createOrUpdate",
-        input: [attribute],
-        lookupField: userId ? "userId" : "anonymousId"
-      },
-      {
-        headers: {
-          Authorization: `Bearer ${token}`,
-          "Content-type": "application/json"
+    const response = marketoResponseHandler({
+      clientResponse: await sendPostRequest(
```

Comment:
We are not using `httpPOST` here ?

Reply:
sendPostRequest is a wrapper on top of httpPost for Marketo.  It returns response which is compatible with reesponsehandler